diff -Naur ./sysrepo-1.3.21/CMakeLists.txt ./sysrepo-1.3.21.PATCH/CMakeLists.txt
--- ./sysrepo-1.3.21/CMakeLists.txt	2019-12-13 13:03:23.000000000 +0000
+++ ./sysrepo-1.3.21.PATCH/CMakeLists.txt	2020-01-17 06:02:50.081299464 +0000
@@ -97,7 +97,7 @@
     if(CMAKE_BUILD_TYPE_LOWER STREQUAL debug)
         set(REPO_PATH "${CMAKE_BINARY_DIR}/repository")
     else()
-        set(REPO_PATH "/etc/sysrepo")
+        set(REPO_PATH "/netconf-yang")
     endif()
 endif()
 set(REPO_PATH "${REPO_PATH}" CACHE PATH "Repository path, contains configuration schema and data files.")
@@ -202,11 +202,11 @@
 
 # sysrepoctl tool
 add_executable(sysrepoctl ${SYSREPOCTL_SRC})
-target_link_libraries(sysrepoctl sysrepo ${LIBYANG_LIBRARIES})
+target_link_libraries(sysrepoctl sysrepo)
 
 # sysrepocfg tool
 add_executable(sysrepocfg ${SYSREPOCFG_SRC})
-target_link_libraries(sysrepocfg sysrepo ${LIBYANG_LIBRARIES})
+target_link_libraries(sysrepocfg sysrepo)
 
 # sysrepo-plugind daemon
 add_executable(sysrepo-plugind ${SYSREPOPLUGIND_SRC})
diff -Naur ./sysrepo-1.3.21/examples/CMakeLists.txt ./sysrepo-1.3.21.PATCH/examples/CMakeLists.txt
--- ./sysrepo-1.3.21/examples/CMakeLists.txt	2019-12-13 13:03:23.000000000 +0000
+++ ./sysrepo-1.3.21.PATCH/examples/CMakeLists.txt	2020-01-17 06:04:02.398376705 +0000
@@ -9,7 +9,7 @@
 
 foreach(app_name IN LISTS examples)
     add_executable(${app_name} ${app_name}.c)
-    target_link_libraries(${app_name} sysrepo yang)
+    target_link_libraries(${app_name} sysrepo)
 endforeach(app_name)
 
 # oven plugin
diff -Naur ./sysrepo-1.3.21/tests/CMakeLists.txt ./sysrepo-1.3.21.PATCH/tests/CMakeLists.txt
--- ./sysrepo-1.3.21/tests/CMakeLists.txt	2019-12-13 13:03:23.000000000 +0000
+++ ./sysrepo-1.3.21.PATCH/tests/CMakeLists.txt	2020-01-17 06:05:03.723315067 +0000
@@ -18,7 +18,7 @@
 
 # set common attributes of all tests
 foreach(test_name IN LISTS tests)
-    target_link_libraries(${test_name} ${CMOCKA_LIBRARIES} sysrepo yang)
+    target_link_libraries(${test_name} ${CMOCKA_LIBRARIES} sysrepo)
     add_test(NAME ${test_name} COMMAND $<TARGET_FILE:${test_name}>)
     set_property(TEST ${test_name} APPEND PROPERTY ENVIRONMENT "MALLOC_CHECK_=3 CMOCKA_TEST_ABORT=1")
 endforeach(test_name)
@@ -26,7 +26,7 @@
 # measure_performance benchmark binary
 set(SR_PERF measure_performance)
 add_executable(${SR_PERF} ${SR_PERF}.c)
-target_link_libraries(${SR_PERF} ${CMOCKA_LIBRARIES} sysrepo yang)
+target_link_libraries(${SR_PERF} ${CMOCKA_LIBRARIES} sysrepo)
 
 # get notification directory
 if(NOTIFICATION_PATH)
