diff -Naur ./libnetconf2-1.1.26/python/CMakeLists.txt ./libnetconf2-1.1.26.PATCH/python/CMakeLists.txt
--- ./libnetconf2-1.1.26/python/CMakeLists.txt	2020-06-02 22:09:38.000000000 +0800
+++ ./libnetconf2-1.1.26.PATCH/python/CMakeLists.txt	2020-07-23 01:19:37.925224981 +0800
@@ -24,5 +24,5 @@
     add_custom_target(pyapidoc COMMAND make -f ${CMAKE_CURRENT_BINARY_DIR}/docs/Makefile html)
     execute_process(COMMAND ${PYTHON} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(plat_specific=True))"
         OUTPUT_VARIABLE PYTHON_MODULE_PATH OUTPUT_STRIP_TRAILING_WHITESPACE)
-    install(CODE "execute_process(COMMAND ${PYTHON} \"${SETUP_PY}\" build -b \"${PYAPI_BUILD_DIR}\" install --install-lib=\$ENV{DESTDIR}/\"${PYTHON_MODULE_PATH}\")")
+    install(CODE "execute_process(COMMAND ${PYTHON} ${SETUP_PY} build -b ${PYAPI_BUILD_DIR} install --install-lib=\$ENV{DESTDIR}/${PYTHON_MODULE_PATH})")
 endif()
diff -Naur ./libnetconf2-1.1.26/python/rpc.c ./libnetconf2-1.1.26.PATCH/python/rpc.c
--- ./libnetconf2-1.1.26/python/rpc.c	2020-06-02 22:09:38.000000000 +0800
+++ ./libnetconf2-1.1.26.PATCH/python/rpc.c	2020-07-23 01:20:40.102638619 +0800
@@ -275,7 +275,7 @@
         goto error;
     }
 
-    rpc = nc_rpc_act_generic(data, NC_PARAMTYPE_FREE);
+    rpc = nc_rpc_act_generic(data, NC_PARAMTYPE_CONST);
     data = NULL;
     if (!rpc) {
         goto error;
diff -Naur ./libnetconf2-1.1.26/python/setup.py.in ./libnetconf2-1.1.26.PATCH/python/setup.py.in
--- ./libnetconf2-1.1.26/python/setup.py.in	2020-06-02 22:09:38.000000000 +0800
+++ ./libnetconf2-1.1.26.PATCH/python/setup.py.in	2020-07-23 01:19:37.925224981 +0800
@@ -12,6 +12,8 @@
                                     "${CMAKE_CURRENT_SOURCE_DIR}/session.h",
                                     "${CMAKE_CURRENT_COURCE_DIR}/rpc.h"
                                    ],
+                           include_dirs=["/arm64-libs/include", "${CMAKE_INCLUDE_PATH}"],
+                           library_dirs=["/arm64-libs/lib", "${CMAKE_LIBRARY_PATH}", "${CMAKE_CURRENT_BINARY_DIR}", "${CMAKE_CURRENT_BINARY_DIR}/.."],
                            libraries=["netconf2"],
                            extra_compile_args=["-Wall", "-I${CMAKE_CURRENT_BINARY_DIR}", "-I${LIBYANG_INCLUDE_DIR}", "-I${LIBSSH_INCLUDE_DIR}" @SSH_DEFINE@ @TLS_DEFINE@],
                            extra_link_args=["-L${CMAKE_CURRENT_BINARY_DIR}/.."],
